### PR TITLE
Add scoreboard-filter to Blitz and Scoring page

### LINKED ISSUE #131: Add missing objective filters
PGMDev/PGM#1118

> Additionally blitz and score modules also support it, and their filters apply for each competitor, so some may show while others do not

`<score>` and `<blitz>` elements support the `scoreboard-filter` but have never been documented as such

### PR DESCRIPTION
**Pull request by TheRealPear**
Resolves #131, building on top of https://github.com/PGMDev/Website/commit/600db7db20e3a2e78d9f87e49c3a49899d58c122

### CODE REVIEW

**Review by Pablete1234 — COMMENTED** [review]
it should note that for both blitz & score, this is a team-scoped dynamic filter, that can filter whether each team should show or not independently

File: docs/modules/objectives/blitz.mdx [hunk 1]
```diff
@@ -27,6 +27,14 @@ Blitz maps should include `<respawn auto="true"/>` to prevent players from "hidi
   | `<join-filter>` | Filters when a player can join the match after it has started. | [Filter](/docs/modules/mechanics/filters) | `never` |
 </div>
 
+##### Blitz Attributes
+
+<div className="table-container">
+  | Attribute | Description | Value |
+  |---|---|---|
+  | `scoreboard-filter` | Determines when the objective should be displayed on the scoreboard.<br />**Note:** This filter is team-scoped, it can filter whether each team should show it or not independently. | [Dynamic Filter](/docs/modules/mechanics/filters#dynamic-filters) |
```
Comment (Pablete1234) [inline docs/modules/objectives/blitz.mdx:35]: Current wording makes it seem like it's the viewer team being filtered, and they get a chance to get it shown or not.

That's not the case, it's the team to display that gets filteredd.

Maybe reword `team should show it or not` to `team should show or not`, there's no separate "it" to show, the team is the "it"

**Review by Pablete1234 — APPROVED** [review]

File: docs/modules/objectives/blitz.mdx [hunk 2]
```diff
@@ -27,6 +27,14 @@ Blitz maps should include `<respawn auto="true"/>` to prevent players from "hidi
   | `<join-filter>` | Filters when a player can join the match after it has started. | [Filter](/docs/modules/mechanics/filters) | `never` |
 </div>
 
+##### Blitz Attributes
+
+<div className="table-container">
+  | Attribute | Description | Value |
+  |---|---|---|
+  | `scoreboard-filter` | Determines when the objective should be displayed on the scoreboard.<br />**Note:** This filter is team-scoped, it can filter whether each team should show or not independently. | [Dynamic Filter](/docs/modules/mechanics/filters#dynamic-filters) |
+</div>
```
Comment (CoWinkKeyDinkInc) [inline docs/modules/objectives/blitz.mdx:36]:
```suggestion
| Attribute | Description | Value |
|---|---|---|
| `scoreboard-filter` | Determines when the objective should be displayed on the scoreboard.<br />**Note:** This filter is team-scoped, it can filter whether each team should show or not independently. | [Dynamic Filter](/docs/modules/mechanics/filters#dynamic-filters) |

```

File: docs/modules/objectives/scoring.mdx [hunk 3]
```diff
@@ -20,6 +20,14 @@ It can also be used to create score boxes that give a player or team a certain a
   | `<box>` | A single score box. | [Score Box](#score-boxes) |
 </div>
 
+##### Score Attributes
+
+<div className="table-container">
+  | Attribute | Description | Value |
+  |---|---|---|
+  | `scoreboard-filter` | Determines when the objective should be displayed on the scoreboard.<br />**Note:** This filter is team-scoped, it can filter whether each team should show or not independently. | [Dynamic Filter](/docs/modules/mechanics/filters#dynamic-filters) |
+</div>
```
Comment (CoWinkKeyDinkInc) [inline docs/modules/objectives/scoring.mdx:29]:
```suggestion
| Attribute | Description | Value |
|---|---|---|
| `scoreboard-filter` | Determines when the objective should be displayed on the scoreboard.<br />**Note:** This filter is team-scoped, it can filter whether each team should show or not independently. | [Dynamic Filter](/docs/modules/mechanics/filters#dynamic-filters) |
```

**Review by CoWinkKeyDinkInc — CHANGES_REQUESTED** [review]

**Review by CoWinkKeyDinkInc — APPROVED** [review]
